### ISSUE TITLE
field lines become obstacles when looking down

### ISSUE THREAD

**Issue by josiahw** [NUbots/robocup#57]
this is only an issue when looking so far down that field lines go off the top of the screen.

**Comment by shannonfenn** [comment]
Yeah this has been known as a problem since the method was first implemented - problem is that I can't think of a fix that won't cause white robots etc. to be unseen as obstacles.

**Comment by josiahw** [comment]
when we are looking below a certain angle, ignore obstacles?

**Comment by shannonfenn** [comment]
Not a complete solution, there are angles at which the lines are still obstacles, but which we still need to see obstacles for avoidance. I'll put this method in for robocup though.

**Comment by StevenNicklin** [comment]
You may be able to use the fact that the lines are white and most of the real obstacles are not to help filter these cases.

**Comment by shannonfenn** [comment]
"Robots may also be colored in aluminimum-like silver, grey or white"

**Comment by josiahw** [comment]
however with the exception of 1 team last year, they were not :)

**Comment by shannonfenn** [comment]
Yeah I just don't want to disable our ability to see them as obstacles for the rare occasion when we see a line that occurs as one - when something like a head angle check might reduce this occurrence to low enough not to be an issue.
